tests/unit/lms/resources/_js_config_test.py

```diff
@@ -22,10 +22,89 @@ def test_it_is_mutable(self, config):
 
         assert config["a_key"] == "a_value"
 
-    def test_enable_content_item_selection_mode(self, js_config):
-        js_config.enable_content_item_selection_mode()
+
+class TestEnableContentItemSelectionMode:
+    def test_it(self, context, js_config):
+        js_config.config["mode"] = "foo"
+
+        js_config.enable_content_item_selection_mode(
+            "test_form_action", "test_form_fields"
+        )
 
         assert js_config.config["mode"] == "content-item-selection"
+        assert js_config.config["formAction"] == "test_form_action"
+        assert js_config.config["formFields"] == "test_form_fields"
```

Comment:
Shouldn't test form fields be a dict? Arguably this shouldn't pass. I don't really believe in the test as documentation thing, but this is misleading too.

Reply:
I've replaced these with `mock.sentinel.form_fields`.

I don't really want to use a dict because, even though it would be a dict really, `JSConfig` just treats it as an opaque value and inserts it into the config.

But strings like `"test_form_fields"` are confusing because they make it look like the value is supposed to be a string.

`mock.sentinel` is a nice way to explicitly say "I'm not implying that this value would actually be a `mock.sentinel` object in production (obviously), I'm explicitly saying that the code treats this thing as an opaque value."